dbt/include/spark/macros/adapters.sql

```diff
@@ -430,3 +431,25 @@
   {% do run_query(sql) %}
 
 {% endmacro %}
+
+{% macro liquid_clustered_cols() -%}
```

Comment:
suggestion , I would add comment here that those macros were added from `dbt-databricks` and from which release , in general it will be better to comment places are from `dbt-databricks` and which parts are custom implementation

Reply:
done